ocaml/xapi/workload_balancing.ml

```diff
@@ -295,6 +295,7 @@ let wlb_request ~__context ~host ~port ~auth ~meth ~params ~handler ~enable_log
   let body = wlb_body meth params in
   let request = wlb_request host meth body auth in
   let pool = Helpers.get_pool ~__context in
+  let verify_cert = Db.Pool.get_wlb_verify_cert ~__context ~self:pool in
```

Comment:
```suggestion
  let verify_cert =
    Db.Pool.(
      tls_verification_enabled ~__context ~self:pool
      || get_wlb_verify_cert ~__context ~self:pool)
  in
```
I think reverting the commit is not enough: the server certificates should be verified when Pool.tls_verification_enabled is enabled, regardless of the value of the get_wlb_verify_cert.

Reply:
This isn't necessary because we do this check in xen-api-libs-transitional already (https://github.com/xapi-project/xen-api-libs-transitional/pull/89/files#r540361707)

I think it's cleaner to keep checks against the global verification flag only inside the stunnel library